src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasRelationSlicer.java

```diff
@@ -632,6 +641,8 @@ private void mergeOverlappingClosedMembers(final Relation relation,
             final int outerIndex = entry.getKey();
             final List<Integer> innerIndices = entry.getValue();
 
+            final CountrySlicingIdentifierFactory lineIdentifierGenerator = new CountrySlicingIdentifierFactory(
```

Comment:
Can you add a quick comment here, I think it's important to call out that the outerIndex is now being used for seeding.

Reply:
done!